static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -118,11 +120,8 @@ const isCompatibleWithEpic = (page: Object): boolean =>
     !page.isMinuteArticle &&
     isArticleWorthAnEpicImpression(page, defaultExclusionRules);
 
-const shouldShowReaderRevenue = (
-    showToContributorsAndSupporters: boolean = false
-): boolean =>
-    (userShouldSeeReaderRevenue() || showToContributorsAndSupporters) &&
-    !config.get('page.shouldHideReaderRevenue');
+const shouldHideReaderRevenue = () =>
```

Comment:
maybe `pageShouldHideReaderRevenue` just to be clear it's about the page, not the user